OsmAnd/src/net/osmand/aidl/IOsmAndAidlInterface.aidl

```diff
@@ -96,6 +96,7 @@ import net.osmand.aidl.mapmarker.RemoveMapMarkersParams;
 
 import net.osmand.aidl.quickaction.QuickActionParams;
 import net.osmand.aidl.quickaction.QuickActionInfoParams;
+import net.osmand.aidlapi.lock.SetLockStateParams;
```

Comment:
This will bring some problems for apps which try to update to new API version because `aidlapi` is another package (newer API) and it might be unimported in some apps.

Reply:
I must confess that I didn't completely understand the concept behind OsmandAidlServiceV2 and OsmandAidlService, since there are a lot of code duplication. 
Is V2 intended to replace the old?

Reply:
Yes, V2 intended to replace old, because there were issues with backwards compatibility for new fields. V2 has a similar structure but uses bundle as the main storage element and it solves some issues with write/read fields order and backwards compatibility for items inside lists.  It is not necessary to add new methods to old api.

Reply:
Ok, now it's clear! Thanks!
So, maybe I'll make a commit removing my adds in the old api, It would be easier to maintain with less code duplication. What do you think?

Reply:
I don't think that removing these changes is necessary, there is much code duplication and one more method will not change the situation. It will be better to test new methods thoroughly instead of removing this adds.